lib/Backend/amd64/LowererMDArch.cpp

```diff
@@ -1643,6 +1653,12 @@ LowererMDArch::GeneratePrologueStackProbe(IR::Instr *entryInstr, IntConstType fr
     // $done:
     //
 
+    if (this->m_func->IsTrueLeaf() &&
```

Comment:
For getter functions, the CallGetter() doesn't probe the stack, we will need isTrueLeaf() due to this.